packages/flat-stores/src/classroom-store/index.ts

```diff
@@ -608,6 +608,29 @@ export class ClassroomStore {
                 this.whiteboardStore.updateAllowDrawing(
                     this.isCreator || whiteboardStorage.state[this.userUUID],
                 );
+                for (const userUUID in diff) {
+                    const user = this.users.cachedUsers.get(this.userUUID);
+                    const enabled = diff[userUUID]?.newValue;
+                    if (user) {
+                        if (this.isCreator) {
+                            if (enabled) {
+                                message.info(
+                                    FlatI18n.t("grant-whiteboard-access", { name: user.name }),
```

Comment:
This user information appears to display the name of the current user, but in fact it should display the name of the object being manipulated, right?

Reply:
Yes, I have a typo above, fixed.

Reply:
Yes, I have a typo above, will fix that.